tokens/tokens.boxShadow.stories.mdx

```diff
@@ -11,24 +11,20 @@ import { Meta } from '@storybook/addon-docs'
     },
   }} />
 
-<!--
-  - https://developer.mozilla.org/de/docs/Web/CSS/box-shadow
-  - https://material.io/design/environment/elevation.html
--->
-
 # Box Shadow tokens
 
 The demosplan-ui BoxShadow tokens variables adds shadow effects around an element's frame.
 
 There are several sizes (from 1 - smallest elevation, to 4 - biggest elevation). Detailed use cases can be found in the token description.
 
 ```
-$dp-box-shadow-1
-$dp-box-shadow-2
-$dp-box-shadow-3
-$dp-box-shadow-4
+$dp-box-shadow-xl
+$dp-box-shadow-lg
+$dp-box-shadow-md
+$dp-box-shadow
 ```
 
+The `$dp-box-shadow-1` to `$dp-box-shadow-4` still work, but will be removed eventually.
```

Comment:
Should this be mentioned in the changelog entry?

Reply:
yes. c3b79a2ceb30ff558f8fffc14d596a2683c0cc97